entertainment_center.py

```diff
@@ -28,9 +28,9 @@ def details(title):
     loaded_json = json.loads(output)
     ID = getTrailerID(title)
     movies.append(media.Movie(loaded_json['Title'],loaded_json['Plot'],loaded_json['Poster'],ID))
-     
 
 for name in names:
     details(name.get('name'))
 
+
```

Comment:
can you revert the changes in this file as they are only space changes just don't want to waste a commit for it.